_studio/shared/umc/codec/vc1_dec/src/umc_vc1_dec_frame_descr.cpp

```diff
@@ -50,9 +50,9 @@ bool VC1FrameDescriptor::Init(uint32_t         DescriporID,
     if (!m_pContext)
     {
         uint8_t* ptr = NULL;
-        ptr += mfx::align_value<uint32_t>(sizeof(VC1Context));
-        ptr += mfx::align_value<uint32_t>(sizeof(VC1PictureLayerHeader)*VC1_MAX_SLICE_NUM);
-        ptr += mfx::align_value<uint32_t>((HeightMB*seqLayerHeader->MaxWidthMB*VC1_MAX_BITPANE_CHUNCKS));
+        ptr += mfx::align2_value<uint32_t>(sizeof(VC1Context));
+        ptr += mfx::align2_value<uint32_t>(sizeof(VC1PictureLayerHeader)*VC1_MAX_SLICE_NUM);
+        ptr += mfx::align2_value<uint32_t>((HeightMB*seqLayerHeader->MaxWidthMB*VC1_MAX_BITPANE_CHUNCKS));
```

Comment:
Do we still need explicit `uint32_t` here?

Reply:
no